cmake/checks/cpu_lsx.cpp

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+#if defined(__loongarch_sx)
+#  include <lsxintrin.h>
+#  define CV_LSX 1
+#endif
```

Comment:
This is build check. No need to introduce `if defined` guards. In case if the check build fails (e.g. no include, no function, etc) the check fails and variable is set to false in CMake script.

Reply:
Thanks for your review, I have remove it.